app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -32,6 +32,13 @@
 
     <h2>Evaluation</h2>
 
+    {% if "change_challenge" in challenge_perms %}
+        <a href="#admin" class="btn btn-danger mb-3">
+            <i class="fas fa-arrow-down"></i>&nbsp;&nbsp;
+            Admin section
```

Comment:
I don't think this is necessary - if tonnes of metrics are included then the admins should hide the metrics and only show the relevant ones, the scrollbar on the right or page down are enough to get to the section.

Reply:
How would new hosts know that that section exists?

Reply:
By looking at the contents of the page.

Reply:
Would they not just assume it's just the details of the evaluation? I would.

Reply:
Removed.